src/librustc_resolve/lib.rs

```diff
@@ -900,7 +900,7 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
             return;
         }
         if self.glob_map.contains_key(&import_id) {
-            self.glob_map[import_id].insert(name);
+            self.glob_map.get_mut(&import_id).unwrap().insert(name);
```

Comment:
In case of library code it generally is, but for code like resolve I think it's fine to just be `unwrap`, backtraces are normally good enough to narrow down the problem to "around here".